src/Controller/EnMarche/ApplicationRequestCandidate/MunicipalChiefSpaceController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace AppBundle\Controller\EnMarche\ApplicationRequestCandidate;
+
+use AppBundle\ApplicationRequest\ApplicationRequestRepository;
+use AppBundle\Entity\ApplicationRequest\ApplicationRequest;
+use AppBundle\Security\Voter\MunicipalChiefVoter;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route(path="/espace-chef-municipal/", name="app_application_request_municipal_chief")
+ *
+ * @Security("is_granted('ROLE_MUNICIPAL_CHIEF')")
+ */
+class MunicipalChiefSpaceController extends AbstractController
+{
+    protected function getApplicationRequests(ApplicationRequestRepository $repository, string $type): array
+    {
+        return $repository->findAllForInseeCodes($this->getUser()->getMunicipalChiefManagedArea()->getCodes(), $type);
+    }
+
+    protected function getSpaceName(): string
+    {
+        return 'municipal_chief';
```

Comment:
Can you use `Enum` for the space name ?

Reply:
I added controller constant for this value, is it ok?

Reply:
:+1: